src/js/containers/aboutTheData/modals/ReportingDifferencesContainer.jsx

```diff
@@ -7,17 +7,17 @@ import React, { useState, useEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
 import { Table, Pagination } from 'data-transparency-ui';
 import { isCancel } from 'axios';
-import { mockAPIReportingDifferences, reportingDifferencesColumns } from 'dataMapping/aboutTheData/modals';
-import { formatReportingDifferencesData } from 'helpers/aboutTheDataHelper';
+import { reportingDifferencesColumns } from 'dataMapping/aboutTheData/modals';
+import { formatReportingDifferencesData, fetchReportingDifferences } from 'helpers/aboutTheDataHelper';
 
 const propTypes = {
-    agencyCode: PropTypes.string,
+    agencyData: PropTypes.object,
```

Comment:
.shape()? Not a very descriptive name. Totally fine but would be a nice to. have to have the .shape